test/scripts/run_test.py

```diff
@@ -158,13 +209,17 @@ def recover_deps():
     subprocess.call(cmd)
 
 
+def recover_unit_test_environment():
+    for module, unit_test in UNIT_TESTS.items():
+        shutil.rmtree(os.path.join(unit_test['path'], 'src/androidTest/jniLibs'))
+
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Run android instrumentation tests.')
```

Comment:
exit result is incorrect in the situation where caselist file for instrumentation test is wrong.

Reply:
done